db/views/searches_v07.sql

```diff
@@ -0,0 +1,45 @@
+WITH defendants AS (
+  SELECT
+    app.id,
+    CASE WHEN app.application_type = 'crm4' THEN
+        (app_ver.application -> 'defendant' ->> 'first_name') || ' ' || (app_ver.application -> 'defendant' ->> 'last_name')
+       ELSE
+        (
+          SELECT (defendants.value->>'first_name') || ' ' || (defendants.value->>'last_name')
+          FROM jsonb_array_elements(app_ver.application->'defendants') AS defendants
+          WHERE defendants.value->>'main' = 'true'
+        )
+       END AS client_name
+  FROM application AS app
+    JOIN application_version AS app_ver
+      ON app.id = app_ver.application_id AND app.current_version = app_ver.version
+)
+SELECT
+  app.id,
+  app_ver.id as application_version_id,
+  app_ver.application ->> 'laa_reference' as laa_reference,
+  app_ver.application -> 'firm_office' ->> 'name' as firm_name,
+  app_ver.application -> 'firm_office' ->> 'account_number' as account_number,
+  app_ver.application ->> 'service_name' as service_name,
+  (app_ver.application -> 'cost_summary' ->> 'high_value')::boolean as high_value,
```

Comment:
The problem with this is that the `high_value` flag has only recently started being set, and it wasn't backfilled, cos I wasn't sure if we'd end up using it. But I think it now makes sense to backfill it, and the good news is it's quite a simple migration (which we only need to do in the app store, because the CW app is going away):

For all submission versions tied to a CRM7 submission where `cost_summary->high_value` is null but `cost_summary` is not null, update to add a high value flag, where the flag is set according to whether `(cost_summary->profit_costs->>gross_cost)::decimal >= 5000`.

For all submission versions tied to a CRM7 submission where `cost_summary` is null, it's harder, because you'll need to create a cost summary hash, and to pass the schema validator, the cost_summary will need to have a bunch of required fields, but we don't know the value of those fields. But that's OK cos we never read them, so you'll want to put in something like this:

```
"cost_summary":{
        "disbursements":{
          "gross_cost":"-1",
          "net_cost":"-1",
          "vat":"-1"
        },
        "profit_costs":{
          "gross_cost":"-1",
          "net_cost":"-1",
          "vat":"-1"
        },
        "travel":{
          "gross_cost":"-1",
          "net_cost":"-1",
          "vat":"-1"
        },
        "waiting":{
          "gross_cost":"-1",
          "net_cost":"-1",
          "vat":"-1"
        },
        "high_value": <whether or not the claim is flagged as high risk>
      }
      
      It's not perfect, in that high risk is not quite the same thing as high value, but it's good enough - that's applying the same fallback that our auto-assignment uses.